src/main/steps/applicant2/upload-your-documents/template.njk

```diff
@@ -78,7 +78,7 @@
     <ul id="filesUploaded" class="govuk-list">
       {% for file in userCase.applicant2DocumentsUploaded %}
         <li class="uploadedFile govuk-!-padding-top-2 govuk-!-padding-bottom-3 govuk-section-break govuk-section-break--visible">
-          {{ file.value.documentFileName }} <a href="/document-manager/delete/{{ file.id }}?_csrf={{ csrfToken }}" class="govuk-link--no-visited-state">{{ delete }}</a>
+          {{ file.value.documentFileName }} <a href="/document-manager/delete/{{ loop.index }}?_csrf={{ csrfToken }}" class="govuk-link--no-visited-state">{{ delete }}</a>
```

Comment:
loop.index? isnt the a way to include it in the for loop? for item, index in ....